tests/braille-specs/la.yaml

```diff
@@ -0,0 +1,2648 @@
+# by aikeo Koomanivong
```

Comment:
It would be great if this test file would reference some official document about the Lao braille code. Does such a document exist? Documentation is important to verify that what is in the test is correct.

Reply:
thanks
Lao braille based on Thai braille
now we try to improve standard for Lao-braille based on UEB and new thai braille generation 
therefore, I do not know about official document to refer except older document from world braille

Reply:
Currently there is no documentation at all, so any reference is good. It doesn't need to be official, and it may be an older document. A link to the [wikipedia page about Thai and Lao braille](https://en.wikipedia.org/wiki/Thai_and_Lao_Braille) is also better than nothing. Just make sure you explain how the table relates to the linked documents. Any useful information about the current state of the standard and the Liblouis table should be added to the comment section at the top of the table.

Reply:
@egli 
@bertfrees 
first of all thanks for help.
would you mind to have some zoom meeting.
I'd like to learn more what I can do next step and how can you might possible supporting us.
please apologize but I'm begin from 0  
I'm looking for your help
thanks